src/components/SeeAll.tsx

```diff
@@ -78,14 +77,12 @@ function SeeAll(props: SeeAllProps) {
                 style={{ textDecoration: 'none' }}
                 to={`${props.match.url}/${key}`}
               >
-                {/* <div className={classes.articlesCard}> */}
-                <StandardCard
+                <ResourceCard
```

Comment:
I'm not too familiar with this part of the codebase, but it seems like the only difference between the `type == 'Playlists'` case and the `type == 'Videos'` case is whether we should render using the data from `lessons` or `videos`, but the actual components and styles that are used are otherwise all the same. Is this correct?

If so, it would be best to not repeat this twice — that way we'd make good use of your refactor which combines all resource types into a single `ResourceCard`! And if we ever add more types in the future, we wouldn't have to copy-paste this code again.

We could maybe have a map of types to the data that should be displayed, like this:

```
const typesToData: Record<string, string[]> = {
    'Playlists': lessons,
    'Videos': videos,
}
```

and then just write the rendering logic once, using `typesToData` to decide which array to render from.

Reply:
Ooh so it's actually between Videos and Instructions (aka Articles) but I'll try that for the Videos + Instructions !!

Reply:
Oh wait, is the `type == 'Playlists'` part a typo then or is that actually want to use for rendering lessons? :O

Reply:
Oh!! hmm so rn we have resources (videos/instructions) uses resourcecard and playlists uses playlistcard -- not sure if we can abstract playlist into resourcecard too since the props are a bit diff